File: templates/java11/template/src/main/java/com/example/HelloWorld.java
package com.example;

import io.nitric.faas.Faas;
import io.nitric.faas.Trigger;
import io.nitric.faas.NitricFunction;
import io.nitric.faas.Response;

public class HelloWorld implements NitricFunction {

    @Override
    public Response handle(Trigger trigger) {
        return trigger.defaultResponse("Hello World".getBytes());
    }

}

File: templates/dotnet/template/src/HelloWorld.cs
using System.Text;
using Nitric.Faas;

namespace NitricTemplate
{
    class HelloWorld : INitricFunction
    {
        public Response Handle(Trigger trigger){
            return trigger.DefaultResponse(
                Encoding.UTF8.GetBytes("Hello World!")
            );
        }
        static void Main(string[] args)
        {
            Faas.Start(new HelloWorld());
        }
    }
}
